editor/src/clj/editor/workspace.clj

```diff
@@ -246,29 +256,66 @@ ordinary paths."
 (defn snapshot-cache [workspace]
   (g/node-value workspace :snapshot-cache))
 
-(defn- is-plugin-file? [resource]
+(defn- is-plugin-clojure-file? [resource]
   (contains? #{"clj"} (resource/ext resource)))
 
-(def dbg-workspace (atom -1))
-
-(defn- find-editor-plugins [workspace]
-  (let [resources (filter is-plugin-file? (g/node-value workspace :resource-list))]
+(defn- find-clojure-plugins [workspace]
+  (let [resources (filter is-plugin-clojure-file? (g/node-value workspace :resource-list))]
     resources))
 
 (defn- load-plugin! [workspace resource]
   ; TODO Handle Exceptions!
   (log/info :msg (str "Loading plugin" (resource/path resource)))
-  (let [plugin (load-string (slurp resource))]
-    (plugin workspace))
+  (let [plugin-fn (load-string (slurp resource))]
+    (plugin-fn workspace))
   (log/info :msg (str "Loaded plugin" (resource/path resource))))
 
 (defn load-editor-plugins! [workspace added]
-  (reset! dbg-workspace workspace)
   (let [added-resources (set (map resource/proj-path added))
-        plugin-resources (find-editor-plugins workspace)
+        plugin-resources (find-clojure-plugins workspace)
         plugin-resources (filter (fn [x] (contains? added-resources (resource/proj-path x))) plugin-resources)]
     (dorun (map (fn [x] (load-plugin! workspace x)) plugin-resources))))
 
+; Determine if the extension has plugins, if so, it needs to be extracted
```

Comment:
Apparently we're supposed to use two semicolons for comments. Single-semicolons are reserved for when comments are at the end of a line of code since their presence at the beginning of a line confuses emacs. Allegedly. 🤷‍♀️